README.md

```diff
@@ -21,9 +21,10 @@ This plugin has been installed, alone with the [General API Plugin](https://gith
 
 ![GitHub Status](docs/images/github-status.png)
 
-By listening to the Jenkins builds, this plugin will automatically publish statuses (pending, in progress, and completed) to GitHub.
+This plugin implements [the status checks feature from Checks API Plugin](https://github.com/jenkinsci/checks-api-plugin#build-status-check) to publish statuses (pending, in progress, and completed) to GitHub.
 
-The status checks can be customized by configuring the "Status Checks Properties" behaviour for you GiHub Source (similar behaviour for Git SCM will be provided soon):
+However, this is not a default feature.
```

Comment:
hmm don't we want it to be the default (like it currently is) by just installing this plugin?

Reply:
oh, I mixed up, so let me make it clear:

1. in checks-api, we set the default value of `skip` to be true, so users won't get logs when no implementation is installed;
2. in github-checks, we set the default value of `skip` to be false, so users can get statuses as long as they installed the implementation, even they didn't add the trait at all?